cryptoki/tests/basic.rs

```diff
@@ -707,6 +703,35 @@ fn is_initialized_test() {
     }
 }
 
+#[test]
+#[serial]
+#[allow(deprecated)]
```

Comment:
Are these suppressions needed? I see nothing deprecated added and the clone warning should also not appear (just wanted to double check).

Reply:
The `deprecated` check can indeed be removed. The clone warning does trigger:

```
error: redundant clone
   --> cryptoki/tests/basic.rs:714:23
    |
714 |     let clone = pkcs11.clone();
    |                       ^^^^^^^^ help: remove this
    |
note: cloned value is neither consumed nor mutated
```

I only clone in that test to show the effect of cloning. Not to do anything useful with the clone.